packages/deploy/src/core/CoreContracts.ts

```diff
@@ -771,163 +769,189 @@ export default class EvmCoreDeploy extends AbstractCoreDeploy<config.EvmCoreCont
 
     //  ========= Home =========
     // Home upgrade setup contracts are defined
-    checklist.assertBeaconProxy(this.data.home, 'Home');
+    checklist.assertBeaconProxy(this.data.home, 'Home beacon proxy is defined');
 
     // updaterManager is set on Home
     const updaterManager = await this.home.updaterManager();
     checklist.equalIds(
       updaterManager,
       this.data.updaterManager,
-      'UpdaterManager',
+      'UpdaterManager is configured on Home',
     );
 
     // state
     const state = await this.home.state();
-    checklist.equals(1, state, `Home state`);
+    checklist.equals(1, state, `Home state is Active`);
 
     // updater
     const homeUpdater = await this.home.updater();
     checklist.equalIds(
       homeUpdater,
       domainConfig.configuration.updater,
-      'Home updater',
+      'Home updater is correctly configured',
     );
 
     // localDomain
     const homeLocalDomain = await this.home.localDomain();
-    checklist.equals(this.domainNumber, homeLocalDomain, `Home local domain`);
+    checklist.equals(
+      this.domainNumber,
+      homeLocalDomain,
+      `Home local domain is correctly configured`,
+    );
 
     // owner
     const homeOwner = await this.home.owner();
     expect(utils.equalIds(homeOwner, this.governanceRouter.address)).to.be.true;
-    checklist.equalIds(homeOwner, this.governanceRouter.address, 'Home owner');
+    checklist.equalIds(
+      homeOwner,
+      this.governanceRouter.address,
+      'Home owner is governance router',
+    );
     //  ========= UpdaterManager =========
     // updater
     const updaterAtUpdaterManager = await this.updaterManager.updater();
     checklist.equalIds(
       updaterAtUpdaterManager,
       domainConfig.configuration.updater,
-      "Updater manager's updater",
+      'UpdaterManager updater is correctly configured',
     );
 
     // owner
     const updaterManagersOwner = await this.updaterManager.owner();
     checklist.equalIds(
       updaterManagersOwner,
       this.governanceRouter.address,
-      "Updater manager's owner",
+      'UpdaterManager owner is governance router',
```

Comment:
Do we want Governance Router capitalised? If not, the suggestion below should be changed to not use capitalised.